test/helpers/default-functions.ts

```diff
@@ -70,52 +79,28 @@ export function defaultFunctions(): void {
 	);
 
 	/**
-	 * Get installed extensions on wiki
+	 * Skip test if extension is not installed (present) on the Wikibase server
 	 */
 	browser.addCommand(
-		'getInstalledExtensions',
-		async ( server: string ): Promise<string[] | undefined> => {
-			const result = await browser.makeRequest(
-				`${server}/w/api.php?action=query&meta=siteinfo&siprop=extensions&format=json`
+		'skipIfExtensionNotPresent',
```

Comment:
Need to update `wdio.d.ts`

Reply:
It is up to date with this though. What am I missing?

Reply:
Updated, my bad